marketplace-elements.js

```diff
@@ -404,6 +404,26 @@
         }),
     });
 
+    document.registerElement('mkt-section', {
+        prototype: Object.create(MktHTMLElement.prototype, {
+            createdCallback: {
+                value: function() {
+                    this.classList.add('mkt-section');
+
+                    if (this.hasAttribute('full')) {
```

Comment:
If you use the attribute classes you can use `this.full` here instead, doesn't matter though.